trade-finance-manager-ui/package.json

```diff
@@ -92,7 +94,6 @@
     "file-loader": "^6.2.0",
     "jest": "29.5.0",
     "node-mocks-http": "^1.13.0",
-    "optimize-css-assets-webpack-plugin": "^6.0.1",
```

Comment:
This package is no longer used. It's been removed in main but looks like it slipped back in when we merged `main` into `main-stbr-tfm-5`. Removing this allows us to not require `--legacy-peer-deps` and do a consistent `npm install` across all the workspace packages.